packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -90,6 +96,10 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
     _localDatabase = context.read<LocalDatabase>();
     _localDatabase.upToDate.showInterest(barcode);
     _daoProductList = DaoProductList(_localDatabase);
+    AnalyticsHelper.trackEvent(
```

Comment:
There is a `TraceableClientMixin` on the other screens.
Wouldn't this be better to track, the opening of the page, with this one instead?

Reply:
@g123k Thank you for your suggestion! I've just pushed it.
I don't know in details how the matomo tracking is going to be used, therefore as it is now the opening of the "add new product" page will be tracked both "my" way and "your" way. Plus the other 5 events, tracked "my" way.